Backend/app/dashboard/CommonAPIs/src/Dashboard/Common/Merchant.hs

```diff
@@ -390,6 +399,28 @@ instance HideSecrets ExotelSmsCfgUpdateReq where
   type ReqWithoutSecrets ExotelSmsCfgUpdateReq = ExotelSmsCfgUpdateTReq
   hideSecrets ExotelSmsCfgUpdateReq {..} = ExotelSmsCfgUpdateTReq {..}
 
+-- WBSms
+
+data WBSmsCfgUpdateReq = WBSmsCfgUpdateReq
+  { passkey :: Text,
+    templSendOtp :: Text,
+    templAltNo :: Text,
+    templWelcome :: Text,
+    url :: BaseUrl
+  }
+  deriving stock (Show, Generic)
+  deriving anyclass (ToJSON, FromJSON, ToSchema)
+
+data WBSmsCfgUpdateTReq = WBSmsCfgUpdateTReq
```

Comment:
here  instead of data newtye  can be used

Reply:
done.

Reply:
why  you have aaded new extra fields, is this is required ??

Reply:
yes it is required... Ideally these fields should be present there.

Reply:
ok